express/src/controllers/create-account.ts

```diff
@@ -168,7 +170,44 @@ export const submitMobileVerificationCode = async function (req: Request, res: R
         res.render('create-account/check-mobile.njk', {mobileNumber: req.session.mobileNumber});
         return;
     }
-    let response = await cognitoClient.verifySmsCode(req.session.authenticationResult?.AccessToken, otp);
-    console.log(response);
-    res.redirect('/');
+    try {
+        let response = await cognitoClient.verifySmsCode(req.session.authenticationResult?.AccessToken, otp);
+        console.log("Got SMS Verification code response")
+        console.log(response);
+
+        const URL = process.env.API_BASE_URL
+        const uuid = randomUUID();
+        const email = req.session.user?.UserAttributes?.filter((attribute: AttributeType) => attribute.Name === 'email')[0].Value;
+        const phone = req.session.mobileNumber;
+
+        let user = {
+                "pk": `user#${uuid}`,
+                "sk": `cognito_username#${req.session.user?.Username}`,
```

Comment:
Can we call it cognito_id instead? Username will be the email address so it might get confusing later.

Reply:
I went with username because that's the actual property used by Cognito but `cognito_id` works fine and I see your point.